_pages/en/it-picture-long-term.md

```diff
@@ -1,25 +1,29 @@
 # IT Vision Picture - Long Term
 
-This is a working document to help create a picture of what will IITB look in the medium term for ESDC.
+This is a working document to help create a picture of what will IITB (Innovation Information and Technology Branch) look in the long term for ESDC (Employment and Social Development Canada).
 
 In the following sections, we're trying to take the same end state and create "views" where different audiences would be looking at ESDC and see different things.
 
-- [Citizens](#citizens)
-- [Private Sector](#private-sector)
-- [IITB-ESDC](#iitb-esdc)
-- [Government Partners](#government-partners)
+- [Citizens](#Citizens)
+  - [Automatic Enrollment and Delivery](#Automatic-Enrollment-and-Delivery)
+  - [Self-Service](#Self-Service)
+  - [Service Centers and Outreach](#Service-Centers-and-Outreach)
+- [Private Sector](#Private-Sector)
+- [IITB-ESDC](#IITB-ESDC)
+  - [Modern Open Organization](#Modern-Open-Organization)
+  - [Strong Culture of Learning](#Strong-culture-of-learning)
 
 ## Citizens
 
-- [Automatic Enrollment and Delivery](#automatic-enrollment-and-delivery)
-- [Self-Service](#self-service)
-- [Service Centers](#service-centers)
-- [IoT Integration](#iot-integration)
+- Cyber security and privacy - citizens have a choice to share their personal data for convenience
+- Generational change: Citizens are used to deal with technologies, they do not face technical illiteracy like 2019.
 
 ### Automatic Enrollment and Delivery
 
 Enrollment and delivery of services is automated by default and wherever possible.
+Canadians are benefiting seamlessly from the federal government services through modern technologies.
```

Comment:
Which modern technologies? 
Which canadians? 
Seamless ... in what way? A hands-off approach where they can receive services without needing to submit forms?

Reply:
Again from @remyb2canada comments, I added it to emphasize the previous statement.

Agreed that modern technology is vague and will remove.

Seamless in the way that enrollment and delivery of services is automated.